en-US/D.xml

```diff
@@ -189,7 +189,13 @@
 			<term>dialog box</term>
 			 <listitem>
 				<para>
-					Refer to the Word Usage chapter of the <citetitle>IBM Style Guide</citetitle> for usage information related to this and similar terms.
+					<emphasis>n.</emphasis> Use "dialog box" (<emphasis>not</emphasis> "dialog", "dialogue", "dialogue box", or "window"), to refer to an element that is displayed for a user to interact with a graphical user interface.
```

Comment:
For concision, I suggest "dialog" instead of "dialog box". Using "dialog" aligns with IBM as well (UI elements > [dialog](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#dialog)). 

This suggestion also aligns with our guidance to not use the element type such as "button": "Click Save" vs "Click the Save button".

Reply:
I really, really do not want to change to "dialog" over "dialog box". This has come up before (I forget when/where) and we decided to stay with dialog box.

This is different from not using "the Save button" IMO. That would have to be a change from "the Save button box" or whatever. Changing the name of an object is not the same as not using the name.